gateway/cli/src/lightning_commands.rs

```diff
@@ -26,6 +28,16 @@ pub enum LightningCommands {
         #[clap(long)]
         description: Option<String>,
     },
+    /// Pay a lightning invoice.
+    PayInvoice {
+        invoice: Bolt11Invoice,
+
+        #[clap(long, default_value_t = 144)]
+        max_delay: u64,
+
+        #[clap(long)]
+        max_fee_msats: Amount,
+    },
```

Comment:
change this comment to clarify that it's the lightning gateway doing the payment , need to distiguish between this action and the pay invoice that fedimint clients are calling

Reply:
Done.